src/make_plots.R

```diff
@@ -0,0 +1,66 @@
+## ---------------------------
+##
+## Script name: make_plots
+##
+## Purpose of script: selecting sites
+##
+## Author: Jack Holbrook (USGS)
+##
+## ---------------------------
+## Notes:
+##     ~
+## ---------------------------
+
+plot_tot <- function(in_monthDat, fileout)
+{
+  #convert((num(n_per_month)))
+  print(in_monthDat)
+  
+  in_monthDat <- readr::read_csv(in_monthDat)
+  #in_monthDat <- as.numeric(in_monthDat$n_per_month)
+  
+  months <- in_monthDat %>% 
+    # uses simple val 27 in absence of costly date check function
+    filter(n_per_month > 27) %>% 
+    subset(month_year < "1995-01-01") %>% 
```

Comment:
Should this be another filter? Not sure why you chose `subset` over `filter`. 

Also, why are you filtering to dates less than 1995? Seems like if you want to count the number of observations in all time (as you do below on line 27), you'd want to keep all data.  Did you mean dates > 1995? I think I would be in favor of doing the initial analysis on all data, and we can use this 1995 criteria later?

Reply:
fixed, has similar solution that uses filter